repl/src/main/scala/com/cloudera/livy/repl/sparkr/SparkRInterpreter.scala

```diff
@@ -62,51 +67,40 @@ object SparkRInterpreter {
     ).r.unanchored
 
   def apply(): SparkRInterpreter = {
-    val executable = sparkRExecutable
-      .getOrElse(throw new IllegalStateException("Cannot find sparkR executable."))
+    val backendTimeout = sys.env.getOrElse("SPARKR_BACKEND_TIMEOUT", "120").toInt
+    val mirror = universe.runtimeMirror(getClass.getClassLoader)
+    val sparkRBackendClass = mirror.classLoader.loadClass("org.apache.spark.api.r.RBackend")
+    val ctor = sparkRBackendClass.getDeclaredConstructor().newInstance()
```

Comment:
`ctor` is a misleading name. This is an instance of the class, not a reference to the class's constructor.